xray/emitter.go

```diff
@@ -43,24 +43,19 @@ func emit(seg *Segment) {
 	}
 
 	for _, p := range packSegments(seg, nil) {
-		if privateCfg.LogLevel() <= log.TraceLvl {
-			b := &bytes.Buffer{}
-			json.Indent(b, p, "", " ")
-			log.Trace(b.String())
-		}
+		b := &bytes.Buffer{}
+		json.Indent(b, p, "", " ")
```

Comment:
hmm, does this introduce a performance regression?

Reply:
hmm it might, going to investigate options either into making it configurable again or removing it.